src/main/java/com/capgemini/jetty/logger/JettyServerCustomizer.java

```diff
@@ -0,0 +1,54 @@
+package com.capgemini.jetty.logger;
+
+import ch.qos.logback.access.jetty.RequestLogImpl;
+import com.netflix.config.DynamicPropertyFactory;
+import org.eclipse.jetty.server.Handler;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.HandlerCollection;
+import org.eclipse.jetty.server.handler.RequestLogHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.embedded.EmbeddedServletContainerFactory;
+import org.springframework.boot.context.embedded.jetty.JettyEmbeddedServletContainerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Class for customizing jetty container and getting access logs based on existing logback-access.xml
+ *
+ * @author Amit Salvi
```

Comment:
Found it - Amit took the starter from this: https://gist.github.com/joakime/27c42782c9a2712a1010.  I'll put a name check in the javadoc, and comment on the gist itself.